media-sound/lollypop/lollypop-1.4.38.ebuild

```diff
@@ -0,0 +1,73 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..12} )
```

Comment:
I suppose you want to use `Closes:` in the commit message to close the relevant bug.